package.json

```diff
@@ -2,48 +2,54 @@
   "name": "sauce-testcafe-runner",
   "version": "0.0.0",
   "description": "Sauce Labs Testcafe runner",
-  "author": "<dev@saucelabs.com>",
+  "author": "<devx@saucelabs.com>",
   "license": "MIT",
   "private": true,
   "bin": "./bin/testcafe",
-  "main": "src/console-wrapper.js",
+  "main": "lib/console-wrapper.js",
   "scripts": {
-    "test": "node src/console-wrapper.js",
-    "lint": "eslint tests/ src/",
+    "build": "tsc",
+    "test": "tsc; node lib/console-wrapper.js",
```

Comment:
Isn't `npm run test` just used as the entrypoint for a docker run? If yes, the bundle already has the built source so I don't think you need to run `tsc`.

Reply:
I removed it in the next PR.